src/wazuh_db/schema_upgrade_v10.sql

```diff
@@ -0,0 +1,38 @@
+/*
+ * SQL Schema for upgrading databases
+ * Copyright (C) 2015, Wazuh Inc.
+ *
+ * Dec 22, 2022
+ *
+ * This program is a free software, you can redistribute it
+ * and/or modify it under the terms of GPLv2.
+*/
+
+CREATE TABLE IF NOT EXISTS _sca_check (
+   scan_id INTEGER REFERENCES sca_scan_info (id),
+   id INTEGER PRIMARY KEY,
+   policy_id TEXT REFERENCES sca_policy (id),
+   title TEXT,
+   description TEXT,
+   rationale TEXT,
+   remediation TEXT,
+   file TEXT,
+   process TEXT,
+   directory TEXT,
+   registry TEXT,
+   command TEXT,
+   `references` TEXT,
+   result TEXT,
+   reason TEXT,
+   condition TEXT
+);
+
+INSERT INTO _sca_check SELECT scan_id, id, policy_id, title, description, rationale, remediation, file, process, directory, registry, command, `references`, CASE WHEN result <> '' THEN result ELSE 'not applicable' END AS result, reason, condition FROM sca_check;
+
+DROP TABLE IF EXISTS sca_check;
+
+ALTER TABLE _sca_check RENAME TO sca_check;
+
+CREATE INDEX IF NOT EXISTS policy_id_index ON sca_check (policy_id);
```

Comment:
I guess we can simplify this operation with something like this:

```suggestion
UPDATE sca_check SET result='not applicable' WHERE status='not applicable';
ALTER TABLE sca_check DROP COLUMN status;
```

This avoids moving all the table data and reindexing it.

Reply:
Done and tested f72ac75df7288169883ff5012c6c9017597b4ff8. Same results as before:
```
sqlite> select status from sca_check;
Error: no such column: status
sqlite> select result,reason from sca_check;
passed|
failed|
not applicable|Could not open file '/var/ossec/etc/FAKE.conf'
```
Thanks for the suggestions!